package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "hof-epp-form",
```

Comment:
Please verify if the name of the app needs to reflect the name of the git repository or hosting platform name